mintpy/objects/cluster.py

```diff
@@ -31,18 +46,37 @@ def get_cluster(cluster_type, **kwargs):
     return cluster
 
 
-def format_walltime_for_cluster(cluster, wall):
-    wall_len = len(wall)
-    formatted_wall = wall
-    if cluster in ['pbs', 'slurm'] and wall_len != 8:
-        if wall_len == 5:
-            formatted_wall = wall + ":00"
+def check_config_name(config_name, cluster_type)
```

Comment:
I would expect any function that starts with the name `check_` to return a boolean indicating whether or not the inputs pass the check, but not to deal with any formatting of the inputs to force obedience to the check. I would rename these functions to `format_{param}`

Reply:
Your suggestion sounds good to me. Go for it please.

Reply:
This is now done in #351.